### PR TITLE
Fixes #136: Responsiveness

### LINKED ISSUE #136: Responsiveness
- Contents are poorly aligned on desktop & iPad, everything is aligned to the left, leaves unnecessary blank space on the right.

image

ipadss

- Lacks padding on phone

iphonexss

------------

Working on it (as GCI task).

### PR DESCRIPTION
**Pull request by Ploypiti**
Fixes #136 | [Preview Link](https://ploypiti.github.io/Ammeg2012.fossasia.org/)

*Iphone 8*
ss
ss2

### CODE REVIEW

File: stylesheets/styles.css [hunk 1]
```diff
@@ -1,248 +1,57 @@
...
-
-header ul {
-  list-style:none;
-  height:40px;
-
-  padding:0;
-
-  background: #eee;
-  background: -moz-linear-gradient(top, #f8f8f8 0%, #dddddd 100%);
-  background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#f8f8f8), color-stop(100%,#dddddd));
```
Comment (0x48piraj) [inline stylesheets/styles.css:112]: Why removed this section ?

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
See the CR.

File: stylesheets/styles.css (hunk 1 above)
Comment (Ploypiti) [reply stylesheets/styles.css:112]: @0x48piraj `stylesheet.css` weren't originally included in head.html, also itself bugged the site w/ overlapping elements.

**Comment by abishekvashok** [comment]
@Ploypiti I have change dmd files to html files now, now it would be easily styleable

**Review by isuruAb — COMMENTED** [review]
now it shows like this
screen shot 2018-01-13 at 6 12 26 pm

**Review by abishekvashok — COMMENTED** [review]
The code affects the nav (drawer icon) its padding is now gone and appears a bit weird. I see you have put in great amount of work. Can you rectify that as well?

**Comment by abishekvashok** [comment]
@Ploypiti i am closing gthis due to inactivity. You can always reopen it when you feel like working on it :)

**Pull request closed without merging**